src/sql/connection.py

```diff
@@ -615,7 +615,7 @@ def get_curr_identifiers(self) -> list:
 
         return identifiers
 
-    def _transpile_query(self, query):
+    def _transpile_query(self, query, read=None):
```

Comment:
document what the read parameter is in the docstring

Reply:
Removed the read parameter, we don't need that anymore